lib/terminal.robot

```diff
@@ -170,5 +172,6 @@ Execute UEFI Shell Command
     Write Bare Into Terminal    ${command}
     Sleep    ${timeout}ms
     Press Enter
+    Sleep    1s
```

Comment:
Is sleep needed?

Reply:
While testing  a couple of times the command have not have a chance to start producing any output before the next keyword was being executed (the one collecting output). The outcome was an empty string as output.